components/textbox/index.ts

```diff
@@ -48,4 +48,9 @@ const mapDispatchToProps = (dispatch: Dispatch<GenericAction>) => ({
     }, dispatch),
 });
 
-export default connect(makeMapStateToProps, mapDispatchToProps, null, {forwardRef: true})(Textbox);
+const connectedTextbox = connect(makeMapStateToProps, mapDispatchToProps, null, {forwardRef: true})(Textbox);
+export default connectedTextbox;
+
+// Attach the Textbox to the window object so that plugins can use it.
+window.Textbox = connectedTextbox;
```

Comment:
I'm new here but maybe its cleaner to add this to the plugins export.js file?

https://github.com/mattermost/mattermost-webapp/blob/master/plugins/export.js

Reply:
Thank you @scastillo, good eye! I can't believe I missed that. 🤦